ledger-core/v2/virtual/object/awaittablefill.go

```diff
@@ -0,0 +1,40 @@
+package object
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/injector"
+)
+
+type SMAwaitTableFill struct {
```

Comment:
It looks like the same as SMAwaitDelegate. Why we need a new SM for semaphore?

Reply:
this point requires a discussion. When it will be possible, we can go for generic approach. For now it is not possible